### PR TITLE
support batch-upload in python-sdk 

### PR DESCRIPTION
**Pull request by magnurud**
- WIP: batch-upload
- first draft
- seconddraft
- thirddraft

### CODE REVIEW

File: las/client.py [hunk 1]
```diff
@@ -558,9 +572,12 @@ def create_document(
         :raises: :py:class:`~las.InvalidCredentialsException`, :py:class:`~las.TooManyRequestsException`,\
  :py:class:`~las.LimitExceededException`, :py:class:`requests.exception.RequestException`
         """
+        find_content_type = not content_type
+        content_bytes, guessed_content_type = parse_content(content, find_content_type)
```
Comment (augustak) [inline las/client.py:576]: Is there a point in not just always guess content type and rather determine how to use it based on 'content_type' arg?
For instance later 'contentType': content_type or guessed_content_type and maybe even print a warning if content_type != guessed_content_type?

```suggestion
        content_bytes, guessed_content_type = parse_content(content)
```

**Review by augustak — COMMENTED** [review]
Add version update

File: las/client.py (hunk 1 above)
Comment (magnurud) [reply las/client.py:576]: It would make the function less efficient if we always guess content type, but it might not be as significant?

Comment (augustak) [reply las/client.py:576]: I think the library looks at the first 10 bytes of the bytes provided. I can't imagine it being any significant?

File: las/client.py [hunk 2]
```diff
@@ -77,26 +85,29 @@ def parse_content(content):
...
     raw = Path(content).read_bytes()
-    return b64encode(raw).decode()
+    content_type = _guess_content_type(raw) if find_content_type else None
+    return b64encode(raw).decode(), content_type
 
 
 @parse_content.register(bytes)
 @parse_content.register(bytearray)
-def _(content):
+def _(content, find_content_type=False):
```
Comment (augustak) [inline las/client.py:96]: Do we need the argument 'find_content_type' anymore?

**Review by augustak — APPROVED** [review]

**Pull request merged**